qiskit/quantum_info/synthesis/two_qubit_decompose.py

```diff
@@ -350,7 +350,8 @@ def __init__(self, gate, basis_fidelity=1.0):
                                   self.decomp3_supercontrolled]
 
     def traces(self, target):
-        """Give the expected traces |Tr(U.Utarget^dag)| for different number of basis gates"""
+        """Give the expected traces :math:`|Tr(U \\cdot Utarget^dag)|` for different number of
```

Comment:
Is there a reason for using `\cdot` here and leaving `.` elsewhere?

Reply:
I might have just missed a spot. If it's somewhere we're using `.` to represent a dot product we should update that.